driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java

```diff
@@ -64,6 +64,8 @@
 
   private static final boolean HOST_METRICS_ENABLED =
       Boolean.getBoolean("com.datastax.driver.HOST_METRICS_ENABLED");
+  static final String DISABLE_QUERY_WARNING_LOGS = "com.datastax.driver.DISABLE_QUERY_WARNING_LOGS";
+  private static final int MAX_QUERY_LOG_LENGTH = 50;
```

Comment:
50 seems a bit short imo. Besides, we already have `com.datastax.driver.core.QueryLogger#DEFAULT_MAX_QUERY_STRING_LENGTH`, maybe we should simply reuse that.

Reply:
Suggestion: maybe you could reuse some of the `QueryLogger` functionality and call `QueryLogger.statementAsString()` to format the statement.